dressForTheWeather/Networking/WeatherNetworking.swift

```diff
@@ -16,9 +16,12 @@ final class WeatherNetworking {
     static let weatherSession = URLSession(configuration: .default)
     static var dataTask: URLSessionDataTask?
     
+    static let darkSkyKey = "e3c6c3f5254ee83b29cf829a4eee5c46"
+    
     // MARK: - Methods
     
-    static func getWeatherFor(latitude: Double, longitude: Double, completion: (@escaping (WeatherData) -> Void)) {
+    static func getWeatherFor(latitude: Double, longitude: Double, success: @escaping (WeatherData) -> Void, failure: @escaping (Error?) -> Void) {
```

Comment:
The Error parameter shouldn't be optional now that we have a closure that will specifically be called if there's an error.